### ISSUE TITLE
LINCC project

### ISSUE THREAD

**Issue by tms-epcc** [lsst-uk/Cross-Match#70]
This covers the half-time FTE LINCC funded project running from Oct '23 to Jan'24.

Github repo for the project is at 
https://github.com/macauff/lsdb_macauff/

**Comment by tms-epcc** [comment]
11/JAN/23
- almost finished, including stretch goals
- there will be a report once this project is finished and possibly a paper 
- there will also be a LINCC talk

**Comment by tms-epcc** [comment]
21/MAR/24
- talk given in w/b 11/MAR/24
- will do a week-long sprint in summer'24

**Comment by tms-epcc** [comment]
02/MAY/24
- there will some further activity in July as previously stated.

**Comment by tms-epcc** [comment]
11/JUL/24
- have also applied for a new LINCC

**Comment by tms-epcc** [comment]
15/AUG/24
- not successful in latest LINCC round, have had email feedback saying not expected same impact as existing LINCC project
- will close the current project in September'24

**Comment by tms-epcc** [comment]
10/SEP/24
- this will no longer be closed in Sept.
- note this is not urgent so will be finished when time is available

**Comment by tms-epcc** [comment]
29/JAN/25
- waiting for Tom to have time available

**Comment by tms-epcc** [comment]
01/MAY/25
- now have a version of the software that will run on LINCC LSDB 
- now need to confirm software works on a suitable platform eg. CSD3 or ANO

**Comment by tms-epcc** [comment]
07/OCT/25
- Tom will attend busy week in w/co 13/OCT in Baltimore

**Comment by tms-epcc** [comment]
17/DEC/25
- attended sprint week
- resulted in PRs 
- funded effort finished hence closed
